docs/types.rst

```diff
@@ -608,6 +607,24 @@ number of bytes, always use one of ``bytes1`` to ``bytes32`` because they are mu
     that you are accessing the low-level bytes of the UTF-8 representation,
     and not the individual characters!
 
+You can initialize an array inline using syntax such as ``string[] myarray = ["a", "b"];``. However, this only works for statically sized memory arrays. For example.
```

Comment:
So actually this example does not work :)

Reply:
@chriseth So it should be…

`string[2] myarray = ["a", "b"];`

?

Which I think makes the two examples kind of the same, so if that is correct I'll consolidate them.

Reply:
Dynamic arrays can be created using `string[] memory myarray  = new string[](7);`

Reply:
And then you have to assign `myarray[0] = ...; myarray[1] = ...; ...`